gateway-js/src/config.ts

```diff
@@ -20,9 +19,7 @@ export type Experimental_DidResolveQueryPlanCallback = ({
   readonly queryPlan: QueryPlan;
   readonly serviceMap: ServiceMap;
   readonly operationContext: OperationContext;
-  readonly requestContext: GraphQLRequestContextExecutionDidStart<
```

Comment:
**nitpick** I think this is only a formatting change, which might be reverted if you format this section of code with prettier.